Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 
 WORKDIR /app
 ADD . .
-RUN ./gradlew --no-daemon assemble
+RUN ./gradlew --no-daemon assemble -x test
```

Comment:
the issue was that the coverage was deleting the libraries as part of the docker build